README.md

```diff
@@ -12,9 +12,36 @@
 - Run and print output or show resulting JavaScript
 - Multiple view modes: code, output or both
 - Persistent session
-- Load PureScript code from Github Gists
+- Load PureScript code from GitHub Gists or repository files
 
-### Which Libraries are Available?
+### Control Features via the Query String
+
+Most of these features can be controlled not only from the toolbar, but also using the [query parameters](https://en.wikipedia.org/wiki/Query_string):
+
+- **Load From GitHub Repo**: Load PureScript code from a GitHub repository using the `github` parameter
+    - Example: `github=/purescript/trypurescript/master/client/examples/Main.purs` will load the code from this file. Note: the file should be a single PureScript module with the module name `Main`.
```

Comment:
Only this section is changed; the rest of this diff is because I swapped this section to be higher (as it's more relevant to users than the "How to update the package set" section)